.github/workflows/update-db-csv-files.yml

```diff
@@ -0,0 +1,70 @@
+name: Update DB files
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Install system dependencies
+        run: |
+          sudo add-apt-repository ppa:deadsnakes/ppa
+          sudo apt-get update
+          sudo apt-get install -y pip python3.10-venv libcurl4-openssl-dev
+
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          persist-credentials: false
+          fetch-depth: 0
+
+      - name: Create Google Service credentials file
+        uses: jsdaniell/create-json@v1.2.3
+        with:
+          name: "service_account.json"
+          json: ${{ secrets.GC_JSON }}
```

Comment:
Similar to previous comment, Sage Monorepo hosts multiple products. We need a way to group secrets, either with a prefix or using an Google secret environment. Also, I would recommend using "GOOGLE_CLOUD" instead of "GC" to make it more descriptive.